editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/index.tsx

```diff
@@ -0,0 +1,158 @@
+}));
+
+export const EXAMPLE_COLOUR = "#007078";
+
+type SettingsFormProps = {
+  legend: string;
+  description: React.ReactElement;
+  input: React.ReactElement;
+  formik: FormikProps<any>;
+  preview?: React.ReactElement;
+};
+
+const useTeam = () => {
+  const [team, setTeam] = useState<Team>({} as Team);
+
+  useEffect(() => {
+    const fetchTeam = async () => {
+      try {
+        const fetchedTeam = await useStore.getState().fetchCurrentTeam();
+        if (!fetchedTeam) throw Error("Unable to find team");
+        setTeam(fetchedTeam);
+      } catch (error) {
+        console.error("Error fetching team:", error);
+      }
+    };
+
+    fetchTeam();
+  }, []);
+
+  return team;
+};
+
+export const SettingsForm: React.FC<SettingsFormProps> = ({
+  formik,
+  legend,
+  description,
+  input,
+  preview,
+}) => {
+  return (
+    <EditorRow background>
+      <form onSubmit={formik.handleSubmit}>
+        <InputGroup flowSpacing>
+          <InputLegend>{legend}</InputLegend>
+          {description}
+          {input}
+        </InputGroup>
+        {preview && (
+          <Box>
+            <Typography variant="h4" my={1}>
+              Preview:
+            </Typography>
+            {preview}
+          </Box>
+        )}
+        <ErrorWrapper
+          error={getIn(formik.errors, "primaryColour")}
+          id="design-settings-theme-error"
+        >
+          <Box>
+            <Button type="submit" variant="contained" disabled={!formik.dirty}>
+              Save
+            </Button>
+            <Button
+              onClick={() => formik.resetForm()}
+              type="reset"
+              variant="contained"
+              disabled={!formik.dirty}
+              color="secondary"
+              sx={{ ml: 1.5 }}
+            >
+              Reset changes
+            </Button>
+          </Box>
+        </ErrorWrapper>
+      </form>
+    </EditorRow>
+  );
+};
+
+const DesignSettings: React.FC = () => {
+  const isUsingFeatureFlag = hasFeatureFlag("SHOW_TEAM_SETTINGS");
+  const team = useTeam();
+  const [open, setOpen] = useState(false);
+
+  const handleClose = (
+    _event?: React.SyntheticEvent | Event,
+    reason?: string,
+  ) => {
+    if (reason === "clickaway") {
+      return;
+    }
+
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <EditorRow>
+        <Typography variant="h2" component="h3" gutterBottom>
+          Design
+        </Typography>
+        <Typography variant="body1">
+          How your service appears to public users
+        </Typography>
+      </EditorRow>
+      {!isUsingFeatureFlag ? (
+        <EditorRow>
+          <FeaturePlaceholder title="Feature in development" />{" "}
+        </EditorRow>
+      ) : (
+        <>
+          <ThemeAndLogoForm team={team} onSuccess={() => setOpen(true)} />
+          <ButtonForm />
+          <TextLinkForm />
+          <FaviconForm />
+          <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
```

Comment:
If we're happy with this approach of using `Snackbar` within the Editor for notifications we might want to move this to a higher level and set up a service to call it in various child components as opposed to repeating it in multiple places.

Reply:
Snackbar works really well in this scenario!